samples/trumpify/trumpify.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import threading
+import time
+
+sys.path.append(os.path.abspath('../../'))
+
+from samples.trumpify.twitter_client import TwitterClient
+from daemo.client import DaemoClient
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
+TW_CONSUMER_SECRET = ''
+TW_ACCESS_TOKEN = ''
+TW_ACCESS_TOKEN_SECRET = ''
+
+INPUT_TWITTER_NAME = 'HillaryClinton'
+MINUTES = 60
+FETCH_INTERVAL_MIN = 5
+TWEET_COUNT = 10
+MONITOR_INTERVAL_MIN = 60
+
+twitter = TwitterClient(
```

Comment:
if we move the creds into the util package, then we don't need these being sent around here